ConfigSpace/forbidden.pyx

```diff
@@ -218,25 +218,19 @@ cdef class MultipleValueForbiddenClause(AbstractForbiddenClause):
 
 
 cdef class ForbiddenEqualsClause(SingleValueForbiddenClause):
-    """
-    A ForbiddenEqualsClause
+    """A ForbiddenEqualsClause
 
     It forbids a value from the value range of a hyperparameter to be
     *equal to* ``value``.
 
-    Example
-    -------
+    .. code:: python
 
-    >>> cs = CS.ConfigurationSpace()
-    >>> a = CSH.CategoricalHyperparameter('a', [1,2,3])
-    >>> cs.add_hyperparameters([a])
-    [a, Type: Categorical, Choices: {1, 2, 3}, Default: 1]
+        cs = ConfigurationSpace({"a": [1, 2, 3]})
 
-    It forbids the value 2 for the hyperparameter *a*
+        # forbids the value 2 for the hyperparameter *a*
+        forbidden_clause_a = ForbiddenEqualsClause(cs["a"], 2)
 
-    >>> forbidden_clause_a = CS.ForbiddenEqualsClause(a, 2)
-    >>> cs.add_forbidden_clause(forbidden_clause_a)
-    Forbidden: a == 2
+        cs.add_forbidden_clause(forbidden_clause_a)
```

Comment:
Following up on my questions from above: the doctest is removed from here, is that on purpose?

Reply:
See above